Mathlib/LinearAlgebra/PiTensorProduct.lean

```diff
@@ -314,7 +314,7 @@ variable {R}
 
 -- mathport name: «expr⨂ₜ[ ] , »
 /-- pure tensor in tensor product over some index type -/
-notation3:100"⨂ₜ["R"] "(...)", "r:(scoped f => tprod R f) => r
+notation "⨂ₜ[" R "]" x => tprod R x
```

Comment:
```lean
notation3:100"⨂ₜ["R"] "(...)", "r:(scoped f => tprod R f) => r
```
cause a build error, not sure if this fix the error. Perhaps someone who knows about notation can fix this